os/mm/mm_heap/mm_partition_mgr.c

```diff
@@ -74,6 +74,27 @@ struct mm_part_s g_mm_parts[CONFIG_NUM_APPS];
 /****************************************************************************
  * Private Functions
  ****************************************************************************/
+#define MINSIZEBITS     5
+#define MAXSIZEBITS     32
+/****************************************************************************
+ * Name : get_mpu_region_size
+ *
+ * Description:
+ *   Return a size value which is a power of two as per MPU restrictions
+ *
+ * Returned value:
+ *   Size which is power of two
+ ***************************************************************************/
+static size_t get_mpu_region_size(size_t sz)
```

Comment:
This is duplicated code. (os/binfmt/libelf/libelf_load.c)
How about making macro with which I mentioned in libelf_load.c?